Translators/WZDx/main.py

```diff
@@ -0,0 +1,84 @@
+import functions_framework
+import json
+import copy
+import logging
+import os
+from Translators.WZDx.request_wrapper import get_rsu_request, get_sdw_request
+from Translators.WZDx.tim_generator import generate_tim
+
+log_level = os.environ.get('LOGGING_LEVEL', 'INFO')
+logging.basicConfig(format='%(levelname)s:%(message)s', level=log_level)
+
+
+def update_sat_region_name(request, tim_body):
+    new_tim = copy.deepcopy(tim_body)
+    region_name = new_tim['dataframes'][0]['regions'][0]['name']
+    region_name = region_name.replace(
+        'IDENTIFIER', f"SAT_{request['sdw']['recordId']}")
+    new_tim['dataframes'][0]['regions'][0]['name'] = region_name
+    return new_tim
+
+
+def update_rsu_region_name(request, tim_body):
+    new_tim = copy.deepcopy(tim_body)
+    region_name = new_tim['dataframes'][0]['regions'][0]['name']
+    region_name = region_name.replace(
+        'IDENTIFIER', f"RSU_{request['rsus'][0]['rsuTarget']}")
+    new_tim['dataframes'][0]['regions'][0]['name'] = region_name
+    return new_tim
+
+
+def translate(wzdx_geojson):
+    tims = []
+    # if no RSUs found, drop that one
+    for feature in wzdx_geojson["features"]:
+        tim_body = generate_tim(feature)
+        if tim_body is not None:
+            sdx_request = get_sdw_request(feature["geometry"])
+            sdx_tim = {
+                "request": sdx_request,
+                "tim": update_sat_region_name(sdx_request, tim_body)
+            }
+            tims.append(sdx_tim)
+
+            rsu_request = get_rsu_request(feature)
+            if rsu_request is not None:
+                rsu_tim = {
+                    "request": rsu_request,
+                    "tim": update_rsu_region_name(rsu_request, tim_body)
+                }
+                tims.append(rsu_tim)
+    return tims
+
+
+@functions_framework.http
+def translateWzdxTIM(request):

```

Comment:
Should specify what version(s) of the WZDx message you support

Reply:
Where/how do you think this would be best represented? I've got a note in the limitations section of the README saying it's currently only 4.1 spec but if there's a better place for this I'll add it there as well.

Reply:
My mistake, I was looking at the README at the root. Can you remove that README or copy yours there? It's a little confusing

Reply:
I'll update the root README to link to the other one. I figure there will be additional TIM tools besides the WZDx translator and each should have a separate README as a unique project.

Reply:
Updated.